### PR TITLE
feat(desktop): add generated licenses (js)

### PR DESCRIPTION
**Pull request by doums**
add licenses for all project JS dependencies

### CODE REVIEW

File: nym-vpn-desktop/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "fmt": "prettier --write --ignore-unknown \"**/*\"",
     "fmt:check": "prettier --check --ignore-unknown \"**/*\"",
     "tscheck": "tsc --noEmit",
-    "tauri": "tauri"
+    "tauri": "tauri",
+    "gen:licenses": "npx license-checker-rseidelsohn --json --customPath license-fmt.json > public/licenses-js.json"
```
Comment (octol) [inline nym-vpn-desktop/package.json:26]: what does this do?

Comment (octol) [reply nym-vpn-desktop/package.json:26]: oh ok I think I get it now

**Review by octol — APPROVED** [review]
The CI changes seem good to me, the other changes I can't really comment on

File: nym-vpn-desktop/package.json (hunk 1 above)
Comment (doums) [reply nym-vpn-desktop/package.json:26]: It generates the JS licenses data as a json file so `public/licenses-js.json`

**Pull request merged**